pkg/cmd/server/origin/master.go

```diff
@@ -588,9 +594,6 @@ func (c *MasterConfig) RunAssetServer() {
 			server.TLSConfig = &tls.Config{
 				// Change default from SSLv3 to TLSv1.0 (because of POODLE vulnerability)
 				MinVersion: tls.VersionTLS10,
-				// Populate PeerCertificates in requests, but don't reject connections without certificates
-				// This allows certificates to be validated by authenticators, while still allowing other auth types
-				ClientAuth: tls.RequestClientCert,
```

Comment:
Don't we need `ClientAuth` to be set to do any sort of validation from the cert pools?